Description: Fix SOVERSION of the shipped library
 Changing it based on some other package's version is not reliable
 .
Author: Balint Reczey <balint@balintreczey.hu>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,8 +14,8 @@
 
 set(kodiplatform_NAME kodiplatform)
 set(kodiplatform_DESCRIPTION "Kodi add-on platform support library")
-set(kodiplatform_VERSION_MAJOR ${APP_VERSION_MAJOR})
-set(kodiplatform_VERSION_MINOR ${APP_VERSION_MINOR})
+set(kodiplatform_VERSION_MAJOR 17)
+set(kodiplatform_VERSION_MINOR 0)
 set(kodiplatform_VERSION_PATCH 0)
 
 if(NOT WIN32)
@@ -41,7 +41,7 @@
 add_library(kodiplatform ${SOURCES} ${PLAT_SOURCES})
 target_link_libraries(kodiplatform ${kodiplatform_LIBRARIES} ${p8-platform_LIBRARIES})
 set_target_properties(kodiplatform PROPERTIES VERSION ${kodiplatform_VERSION_MAJOR}.${kodiplatform_VERSION_MINOR}.${kodiplatform_VERSION_PATCH}
-                                              SOVERSION ${kodiplatform_VERSION_MAJOR}.0)
+                                              SOVERSION ${kodiplatform_VERSION_MAJOR})
 
 install(TARGETS kodiplatform DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES src/util/XMLUtils.h

